BoxContentSDK/BoxContentSDK/Requests/BOXFolderItemsRequest.m

```diff
@@ -179,6 +179,7 @@ - (void)performPaginatedRequestWithCached:(BOXItemArrayCompletionBlock)cacheBloc
                                   inRange:(NSRange)range
 {
     BOXFolderPaginatedItemsRequest *paginatedRequest = [[BOXFolderPaginatedItemsRequest alloc] initWithFolderID:self.folderID inRange:range];
+    paginatedRequest.cacheClient = self.cacheClient;
```

Comment:
Is the behavior correct? Does it call the cache client with the expected sub range?